lib/email.ts

```diff
@@ -56,4 +56,39 @@ export const sendVerificationEmail = async (user) => {
     <br/>
     Your snoopForms Team`,
   });
-};
\ No newline at end of file
+};
+
+export const sendForgotPasswordEmail = async (user) => {
+  const token = jwt.sign(
+    { id: user.id },
+    serverRuntimeConfig.nextauthSecret + user.email,
+    {
+      expiresIn: "1d",
+    }
+  );
+  const verifyLink = `${
+    serverRuntimeConfig.nextauthUrl
+  }/auth/reset-password?token=${encodeURIComponent(token)}`;
+  await sendEmail({
+    to: user.email,
+    subject: "Reset your snoopForms password",
+    html: `Someone has requested a link to change your password. You can do this through the link below:<br/>
```

Comment:
**You have requested to change your password**  instead of **Someone has requested a link to change your password**